### PR TITLE
Contact me section complete

### PR DESCRIPTION
**Pull request by ometman**
- Added footer with contact me section 
- Added heading, and introduction text in the contact me section.
- Added contact form in the contact me section.
- Modified styles.css to properly style the contact me section.
-  Tested the contact form with  formspree account.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -284,5 +282,50 @@ <h1 class="about-primary-text">
...
+                            <input id="email" class="name-email email-pos" name="user_email" type="email" placeholder="natajayanj14@gmail.com" required/>
+                        </li>
+                        <li class="contact-form-pos">
+                            <textarea id="message" class="message message-pos" name="user_message" maxlength="500" type="text" required placeholder="Write your message here"></textarea>
+                            </textarea>
+                        </li>
+                        <li class="submit-btn-pos">
+                            <button id="submit-btn" class="submit-btn" name="submit" type="submit"> Get In touch </button>
+                        </li>
+                    </ul>
```
Comment (ichala) [inline index.html:315]: - [ ] Kindly remove the empty space on the right side , to match the Figma design 

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi  Team
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Validations Works  ✔️ 
- Used flex to align elements✔️ 
- No linter errors ✔️ 



 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ometman) [reply index.html]: Updated styles.css and  and index.html to remove space on the side.

Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @ometman 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ The Form Submitted Successfully 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ometman** [comment]
Thanks for the review. Following with desktop design in another branch.

**Comment by ometman** [comment]
Mobile-first design completed and approved.

**Review by topeogunleye — COMMENTED** [review]
Hi @ometman, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
## Merged and Deleted Pull Request ⚠️

- Pull request is merged and deleted. Please do not submit a merged and deleted pull request for review. Kindly revert your merging and deletion or create a new repo for your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Comment by ometman** [comment]
The contact-me-section branch has been restored.

**Comment by topeogunleye** [comment]
> The contact-me-section branch has been restored.

@ometman, I kindly suggest that you submit again for a rereview. Invalid code Reviews do not count as reviews and your number of reviews left will remain intact.

**Pull request merged**